### PR TITLE
iops: init at 0.1

### PR DESCRIPTION
**Pull request by davidak**
successfully built on NixOS x86_64.

I asked Folkert for the license via E-Mail (GPL2).

### CODE REVIEW

File: pkgs/tools/system/iops/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 = "1knih6dwwiicycp5ml09bj3k8j7air9bng070sfnxwfv786y90bz";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp iops $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "measure I/O operations per second of a storage device";
```
Comment (jagajaga) [inline pkgs/tools/system/iops/default.nix]: Please start with a capital letter.

**Comment by davidak** [comment]
@jagajaga i have fixed that.

**Comment by lethalman** [comment]
Lol nice, I've written a very similar tool in C. I wonder when it does make sense to flush the cache when reading from raw devices. Perhaps to avoid something like bcache?

**Comment by davidak** [comment]
@lethalman i think to get the read performance from disk and not from cache.

**Comment by lethalman** [comment]
@davidak there's no cache when dealing with raw device blocks

**Pull request merged**